RunScripts/rdlint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# PATH in new macOS versions doesn't include path below where rdlint is installed
+export PATH=$PATH:/usr/local/bin
+
+if which rdlint > /dev/null; then
+    rdlint swiftlint
+else
+    echo "error: rdlint is not installed. Follow instructions here: https://github.com/readdle/rdlint"
```

Comment:
I don't have access to the repository with the RdLint. Can you please give me access to the repo, so I can install it, before running the script?

Reply:
Sure, I'm working on it )

Reply:
done, please check the link again